doc/source/Tutorials/GettingStarted/index.md

```diff
@@ -1,407 +1,499 @@
+
+2. Click and drag the nodes to a lower position in the graph.
+
+    ![The graph with some added space](images/mainRenderSettingsWithGap.png "The graph with some added space")
+
+
+### Adding a Shader ###
+
+Now that you have more space, it's time to add some shading nodes:
+
+1. Below and to the left of the Group node, create a Disney Material node (_Appleseed_ > _Shader_ > _Material_ > *As_Disney_Material*).
+
+2. In the _Node Editor_, give the shader some reflective surface properties:
+	- Set the Specular plug to `0.6`.
+	- Set the Roughness plug to `0.35`.
+
+3. With the Disney Material node still selected, create a ShaderAssignment node (_Scene_ > _Attributes_ > _ShaderAssignment_).
+
+4. Click and drag the ShaderAssignment node onto the connector connecting the Group and StandardOptions nodes. The ShaderAssignment_node will be interjected between them.
+
+    ![The ShaderAssignment and Disney Material nodes](images/graphEditorFirstShaderNodes.png "The ShaderAssignment and Disney Material nodes")
+
+Note that the input and output plugs on the Disney Material node flow from left to right. This is because in Gaffer:
+
+> Important :
+> Scenes flow from top to bottom, and shaders flow from left to right.
+
+In your newly shaded graph, the ShaderAssignment node takes the material flowing in from the left and assigns it to Gaffy's geometry flowing in from the top. Now that Gaffy has received a proper material assignment, you will need to light the scene.
+
+
+### Adding an Environment Light ###
+
+Lights, like geometry and cameras, are objects that need to be combined with the main scene. For simplicity, you should add a global environment light:
+
+1. Create a PhysicalSky node (_Appleseed_ > _Environment_ > _PhysicalSky_).
+
+2. Place it next to the Camera node.
+
+3. In the _Node Editor_, adjust the node's angle and brightness:
+	- Set the Sun Phi Angle plug to `100`.
+	- Set the Luminance plug to `2.5`.
+
+4. Connect the PhysicalSky node's _out_ plug to the Group node's _in3_ plug.
+
+![A new environment light node](images/graphEditorEnvironmentLightNode.png "A new environment light node")
+
+For the light to take effect, you will need to enable environment lighting in the AppleseedOptions node:
+
+1. Select the AppleseedOptions node.
+
+2. In the _Node Editor_, expand the _Environment_ section.
+
+3. Toggle the switch next to the Environment Light plug to enable it.
+
+4. Type `/group/light` into the plug's field.
+
+The interactive render will now be in the process of updating, and you will be able to see Gaffy with some basic shaders and lighting.
+
+![The first render with a shader and lighting](images/viewerRenderOneShader.png "The first render with a shader and lighting")
+
+
+### Adding Textures ###
+
+As Gaffy is looking a bit bland, you should assign some textures to the shader:
+
+1. Create an Appleseed Color Texture node (_Appleseed_ > _Shader_ > _Texture2d_ > *As_color_texture*).
```

Comment:
Since Gaffer now ships its own OSL shader library, should we try to use those over the as_ versions when possible? @danieldresser ?

Reply:
Gaffer's OSL shader library is a mess.  I would say it's true that it would be great to use it ... but I don't think we've actually got a texture node included current :(